packages/types/schema/BasicProfile.md

```diff
@@ -0,0 +1,21 @@
+# BasicProfile
+
+A **BasicProfile** contains basic profile information about a parcel.
+
+## IPLD Schema
+
+```ipldsch
+type BasicProfile struct {
+	name optional String
+	url optional String
+}
+```
+
+## Browser Support
+
+| Property | [Geo Web Cadastre](https://github.com/Geo-Web-Project/cadastre) | [GeoWeb.app](https://geoweb.app) |
+| -------- | --------------------------------------------------------------- | -------------------------------- |
+| `name`   | ✅                                                              | ✅                               |
+| `url`    | ✅                                                              | ❌                               |
```

Comment:
Isn't the `url` here supported by the browser? I think we use it for the "Web Content" screen.